tartufo/types.py

```diff
@@ -27,6 +27,11 @@ class GitOptions:
     branch: Optional[str]
 
 
+@dataclass
+class FolderOptions:
+    pattern: Optional[str] = "*"
```

Comment:
Default values for options should be set in the command option, rather than in these classes; that way, it is fully transparent and obvious to the user.

Reply:
This enables the CLI and the programmatic interfaces maintain the same defaults.

Reply:
I prefer to write all python code as an API and CLI is but one consumer of that API.